alfresco-googledrive-repo-base/src/main/java/org/alfresco/integrations/google/docs/service/GoogleDocsServiceImpl.java

```diff
@@ -1049,7 +1047,34 @@ private void getDriveFileContent(Credential credential, NodeRef nodeRef, String
                     "Unable to create activity entry: " + jsonException.getMessage(), jsonException);
         }
     }
-    
+
+    private InputStream exportGoodleDriveFile(final File file, final String mimetype,
+        final Drive drive, final NodeRef nodeRef) throws IOException
+    {
+        final Object versionType = fileFolderService.getFileInfo(nodeRef).getProperties()
+                                                    .get(PROP_VERSION_TYPE);
+
+        final String editorURL = String.valueOf(fileFolderService
+            .getFileInfo(nodeRef).getProperties().getOrDefault(PROP_EDITORURL, ""));
+
+        // Different export mechanisms depending on the GD file mimetype (in GD, not Alfresco)
+        if (versionType == null ||
+            isGoogleDriveMimeType(mimetype) ||
+            editorURL.contains("://docs.google.com/document"))
```

Comment:
What about "spreadsheets" (Google Sheets) and "presentation" (Google Slides) in the url ?

Reply:
Fixed.